cbor/idf_component.yml

```diff
@@ -1,4 +1,4 @@
-version: "0.5.4"
+version: "0.6.0"

```

Comment:
Did you try running IDF example with this component version? Any impact on the memory utilization?

Reply:
Also I see float encoding related new source files here, should we add them to our component build list?

Reply:
Yes I tested the cbor example in idf with this version. It worked as expected. However the binary size increased by ~2KB

Reply:
I have also added the float encoding related source files in CMakeList.txt